contract/src/platform-goals/marshal-produce.js

```diff
@@ -0,0 +1,34 @@
+// @ts-check
+import * as marshal from '@endo/marshal';
+import * as patterns from '@endo/patterns';
+
+/**
+ * @typedef {{
+ *   endo1: {
+ *     marshal: typeof import('@endo/marshal');
+ *     patterns: typeof import('@endo/patterns');
+ *   }
+ * }} Endo1Modules
```

Comment:
This is cool! Why `endo1` instead of `endo`?

Reply:
cuz it doesn't have all of endo; just the ones needed by this 1st use case

but that reminds me... I should get input from endo folks such as @kriskowal